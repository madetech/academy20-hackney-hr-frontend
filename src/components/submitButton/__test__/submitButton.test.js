import React from 'react';
import renderer from 'react-test-renderer';
import SubmitButton from '../submitButton';

test('Button is rendered', () => {
  const component = renderer.create(
    <SubmitButton />,
  );
  let test = component.toJSON();
  expect(test).toMatchSnapshot();
  
});