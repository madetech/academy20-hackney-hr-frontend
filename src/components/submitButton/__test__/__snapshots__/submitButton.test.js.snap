// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button is rendered 1`] = `
<input
  className="button--submit"
  type="submit"
  value="Submit"
/>
`;
